types/firefox-webext-browser/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: https://developer.mozilla.org/en-US/Add-ons/WebExtensions
 // Definitions by: Jacob Bom <https://github.com/bomjacob>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.4
+// TypeScript Version: 2.9
```

Comment:
You don't need to version bump with this change - union types are old, and these types should still be compatible with older ts versions.